plugins/LilyPad/LilyPad.cpp

```diff
@@ -604,15 +593,15 @@ void Update(unsigned int port, unsigned int slot) {
 							int id = idList[i] - 0x1104;
 							s[port][slot].buttons[id] = values[i];
 						}
-						if (abs(s[port][slot].sticks[0].vert) <= 48) {
+						if (abs(s[port][slot].buttons[14]) <= 48) {
```

Comment:
Not sure if this also requires the entire negative range of the vert to be included.